docs/e2e/requirements.md

```diff
@@ -0,0 +1,38 @@
+#### Real RP environment requirements
+
+The following environment variables are required for running the e2e tests against the real RP
+
+| Name | Notes |
+|--- | --- |
+| `AZURE_CLIENT_ID` | authentication |
+| `AZURE_CLIENT_SECRET` | authentication |
+| `AZURE_TENANT_ID` | authentication |
+| `AZURE_SUBSCRIPTION_ID` | existing cluster |
+| `AZURE_REGION` | existing cluster location |
+| `RESOURCEGROUP` | existing cluster group |
+
+#### Fake RP environment requirements
```

Comment:
These are all things needed by the fake RP itself, not by the test.  That documentation belongs somewhere else.
In the fake RP case, the test needs the _data/containerservice.yaml for the existing cluster to pull out the credentials with which to connect.

Reply:
If #880 merges as is, the e2e tests will also need a [Real] / [Fake] focus input - this needs to be documented here.